python/nav/util.py

```diff
@@ -509,3 +509,18 @@ def _range_to_str(x, y):
             return str(x)
         else:
             return "{}-{}".format(x, y)
+
+
+def check_log_level(logger, loglevel):
+    """
+    Check that the level of the logger is equal to or below loglevel
+
+    Use before possibly expensive logging operations, like calling logging in
+    a loop or building an expensive object that will be logged and thrown away.
+
+    Avoid this classic mistake:
+
+        logger.info("%s", expensive_function())
+    """
+    level = logger.getEffectiveLevel()
+    return level <= loglevel
```

Comment:
Referring back to my previous review comments: Why do we need this at all? Why not just use `logger.isEnabledFor(loglevel)` ?

Reply:
https://docs.python.org/3/library/logging.html#logging.Logger.isEnabledFor